stats/opentelemetry/e2e_test.go

```diff
@@ -127,12 +95,11 @@ func (s) TestMethodTargetAttributeFilter(t *testing.T) {
 		// Will allow duplex/any other type of RPC.
 		return str != "/grpc.testing.TestService/UnaryCall"
```

Comment:
I believe the generated code defines a const for the method names. Could we use that instead?

Reply:
Ah, good point. Switched.